src/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+</head>
+<body>
+    <h1>Leaderboard</h1>
+    <div id="main-container">
+        <section id="recent-scores-main-container">
+            <div id="recent-scores-top">
+                <h3>Recent scores</h3>
+                <span id="refresh-item">Refresh</span>
+            </div>
+            <ul id="scores-list">
+                <li class="list-element">Name: 100</li>
+                <li class="list-element">Name: 50</li>
+                <li class="list-element">Name: 60</li>
+                <li class="list-element">Name: 70</li>
+                <li class="list-element">Name: 30</li>
+                <li class="list-element">Name: 10</li>
+            </ul>
+        </section>
+        <section id="add-score-container">
+            <h3>Add your score</h3>
+            <form id="new-score-form">
+                <input type="text" placeholder="Your name">
+                <input type="text" placeholder="Your score">
+                <input id="submit-btn" type="submit" value="Submit">
+            </form>
+        </section>
+    </div>
```

Comment:
- [ ] Please, inside the body on `index.html` consider to use a root element like `<div id="root"></div>` (this element is the only one who need to be inside index.html body, then, on js code (on index.js) create all element required for this project. 🐶